### PR TITLE
Update constitution.rst

### PR DESCRIPTION
**Pull request by Sonia-Kang**
4 amendments of constitution at 2020 S2 GM

### CODE REVIEW

File: constitution.rst [hunk 1]
```diff
@@ -101,7 +101,7 @@ The Executive
    11. A Technology Events Coordinator (Cyber Security)
    12. And up to a further seven Executive Members, comprising:
 
-       1. An additional Industry Liaison
+       1. Two additional Industry Liaison
```
Comment (auscompgeek) [inline constitution.rst]: Grammar?

File: constitution.rst [hunk 2]
```diff
@@ -451,6 +460,35 @@ Elections
     Executive Member is dismissed from the Executive or resigns
     before the term of office has expired.)
 
+Vacancies and Dismissals
+---------
```
Comment (auscompgeek) [inline constitution.rst]: constitution.rst:464: (WARNING/2) Title underline too short.

**Review by thewrongjames — APPROVED** [review]
Comparing this to the agreed upon amendments it checks out and all seems to be right, so, I approve.

I presume they aren't actually a problem (hence I have approved) but there are some labelling discrepancies that I'd like to run by you @Sonia-Kang before we merge this.

The first ones are just that the clause 4.1 changes in the constitutional amendments coversheet that Barbara sent me on slack (attached (github wouldn't attach .doc though so I did have to re-export, maybe just check it in slack)) use a different numbering system to that in the constitution (they like, use Roman numerals or letters in parts where the actual constitution is numbers all the way down). Is that okay?

The second thing is that the vacancies and dismissals section is labelled as section 15 in the changes document, but in the new compiled version is section 10, and even if we were to move it to the end, unless we put it after the recognition section it would only go as high as section 14. Is that okay?

I don't know if any of this is a problem, I don't know how strictly we have to stick to the changes document. I just thought I should mention them in case.

[Constitutional Amendments Coversheet.pdf](https://github.com/syncs-usyd/syncs-constitution/files/5629144/Constitutional.Amendments.Coversheet.pdf)

**Comment by Sonia-Kang** [comment]
For the numbering differences between the constitutional amendments coversheet by Barbara and the constitution, it should be fine for this time, we will just need to make sure they are the same for future GM/AGM.

I thought section 10 would be a more reasonable place for 'vacancies and dismissals' since it is after the 'election' section.

@thewrongjames These are only small editing problems that shouldn't cause anymore USU admin trouble, but thanks for noticing and mentioning them, I will check with Barbara if we can do the above changes without triggering any additional USU admin process.

**Pull request merged**